.github/workflows/ci.yml

```
@@ -29,103 +26,191 @@
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
 
-      - name: Check package can be packed
-        run: npm pack --dry-run
+      - name: Validate changelogs
+        run: |
+          # Check if source files changed
+          if git diff --name-only ${{ github.event.before }} ${{ github.sha }} | grep -q '^src/'; then
+            echo "Source files changed, checking changelog updates..."
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)